core/node/consensus/src/storage/connection.rs

```diff
@@ -106,6 +106,7 @@ impl<'a> Connection<'a> {
     }
 
     /// Wrapper for `consensus_dal().insert_block_certificate()`.
+    #[tracing::instrument(skip_all, fields(block_number = %cert.message.proposal.number))]
```

Comment:
JFYI: In the recently added traces I've tried to use field names `l1_batch` and `l2_block`, so that it's easy to build consistent filters when you're looking for traces. There is no strict notion right now, but it may be beneficial to follow the same convention everywhere. I'll prepare a doc on tracing a bit later.